### DISCUSSION TITLE
[suggestion] Liaison Paramètre d'une scène

### DISCUSSION THREAD

**Discussion by KenneyClement** [witsa/synapps#703] [Ideas]
Ajouter la possibilité de créer une liaison dans les paramètres d'une scène.

**Comment by manufitoussiwit** [comment]
Salut, 

Il est possible de mettre à jour un paramètre de scène (comme d'ailleurs une propriété dans un composite) par liaison :
La liaison se fait sur la source plutôt que sur le paramètre et on active l'option d'écriture.

Par exemple, disons que nous voulons écrire dans le paramètre `prop1` d'une scène la valeur d'un acteur bouton interrupteur.
Nous réalisons une liaison sur la valeur de bouton interrupteur vers le paramètre `prop1` de la scène et nous activons l'écriture : 
![image](https://github.com/witsa/synapps/assets/35595723/d6af9df0-f88e-4975-97f6-f8aa5bdc985d)
Ainsi lorsque le bouton sera basculé, la valeur de `prop1` va suivre.

Je joins l'exemple : 
```
SYNAPPS-STUDIO-SCENE|{"config":{"key":"scene9","name":"Exemple paramètre liaison","additionalDefs":{"prop1":{"type":"boolean"}}},"leadActor":{"type":"layout/stack","key":"stack1","children":[{"type":"input/switch-button","key":"switch-button1","properties":{"horizontalAlignment":"middle","verticalAlignment":"middle"},"bindings":{"properties.value":{"canWrite":true,"source":"stage@additionals.prop1"}}}]}}
```

**Reply by KenneyClement** [reply]
Donc impossible de lier des ressources dans la partie scène ?

**Reply by manufitoussiwit** [reply]
Par liaison non. 
Par script oui (depuis `onRequestDone` d'un acteur fournisseur de ressource/variable ou `onDidDataStore` d'un fournisseur relatif, on peut écrire la valeur désiré dans le paramètre de scène.)

**Reply by KenneyClement** [reply]
Ok merci
